### ISSUE TITLE
Update images for high DPI screens

### ISSUE THREAD

**Issue by splorp** [splorp/newton-glossary#41]
Create 2x versions of images (especially the bitmapped screenshots) and include either srcset values or natural height and width attributes.

This has been tested on both regular and Retina resolution screens in Chrome.

**Comment by splorp** [comment]
Just over three dozen images have been up-rez’d and added to the `assets/images/` directory. (See list below.)

All of the up-rez’d can be recognized by the addition of a `-2x` suffix.

**Note:** The image paths still need to be updated in the corresponding content files.

```
action-picker-2x.gif
alert-box-2x.gif
alert-box-nothing-is-selected-2x.gif
alert-box-reminder-2x.gif
alert-box-reminder-with-snooze-picker-2x.gif
alert-box-unable-to-print-2x.gif
assist-slip-2x.gif
assist-slip-please-picker-2x.gif
availworks-graph-2x.gif
availworks-text-2x.gif
busy-cursor-2x.gif
caret-picker-2x.gif
caret-picker-with-caret-2x.gif
chatbuddy-2x.gif
classroom-dock-2x.gif
classroom-dock-connect-to-2x.gif
classroom-dock-connect-via-2x.gif
connection-2x.gif
connection-preferences-2x.gif
dash-board-2x.gif
deep-space-newt-2x.gif
deep-space-newt-old-2x.gif
dock-2x.gif
dock-connect-2x.gif
dock-connect-appletalk-2x.gif
dock-connect-irda-2x.gif
dock-connect-modem-2x.gif
dock-connect-serial-2x.gif
fitaly-notepad-2x.gif
fitaly-with-number-pad-2x.gif
fitaly-without-number-pad-2x.gif
folder-slip-folders-2x.gif
folder-slip-stores-2x.gif
folder-slip-stores-folders-2x.gif
hemlock-results-2x.gif
hemlock-search-2x.gif
hypernewt-2x.gif
info-button-2x.gif
ink-spot-2x.gif
inspector-2x.gif
keyboard-equivalent-2x.gif
```

**Comment by splorp** [comment]
This issue was resolved per commit ea2d84a
